dist.ini

```diff
@@ -95,6 +95,8 @@ skip = ^MIME::Lite$
 perl                   = 5.008
 Regexp::Common         = 2013031301
 Config::Tiny           = 0
+Date::Format           = 0
+Date::Parse            = 0
 File::ShareDir         = 0
```

Comment:
Avoided needing Date::__ at all, I was initially concerned about leap seconds throwing the calculations out, but on reading the spec it became clear that unix time doesn't include them, so the calculation is actually really simple.